template/en/default/search/form.html.tmpl

```diff
@@ -43,10 +43,10 @@ var tms = new Array();
 [%# Create an array of products, indexed by the classification #%]
 
 [% nclass = 0 %]
-[% FOREACH c = classification %]
+[% FOREACH classification = classification %]
```

Comment:
Hooray TT. Looks like this change introduces another issue 😅 The Classification list will be empty on the Advanced Search page. `cls` or whatever should work.

Reply:
damnit TT

Reply:
_/me awaiting an update..._